orc8r/cloud/go/mproto/mproto_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mproto_test
+
+import (
+	b64 "encoding/base64"
```

Comment:
Don't alias unless necessary to avoid name conflict etc. So in this case just leave it as `base64`, unlaiased